<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestScreenSharing</class>
 <widget class="QDialog" name="TestScreenSharing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>屏幕录制分享</string>
  </property>
  <widget class="QPushButton" name="addExcludedShareWindow">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>加入到排除列表</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoFps">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>15</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="pauseScreenCaptureBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停 / 恢复屏幕的分享</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectScreenCaptureTarget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>点击更新分享参数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeAllExcludedShareWindow">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>移除所有窗口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>视频帧率：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="right">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="minVideoBitrate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>50</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="screenCaptureSourceType">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="currentText">
    <string>整个桌面</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>某个窗口</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>整个桌面</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="removeExcludedShareWindow">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>从排除列表移除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>39</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>动态调整分辨率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="sdkVersionLabel_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择分享类型：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableHighLight">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>高亮窗口描边</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Left：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="startScreenCaptureButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>开始 / 停止屏幕的分享</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="top">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableCaptureMouse">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>采集鼠标</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sdkVersionLabel_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择要分享目标：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="screenCaptureSources">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="sdkVersionLabel_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>混音音量大小：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableHighPerformance">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>高性能模式</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Right：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>分辨率模式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>视频分辨率：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bottom">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="left">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableAdjustRes">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>允许</string>
   </property>
  </widget>
  <widget class="QLabel" name="sdkVersionLabel_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>指定捕获的区域</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="videoBitrate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>550</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Top：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="resMode">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>横屏分辨率</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>竖屏分辨率</string>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="screenCaptureMixVolumeSlider">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>最低码率：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="videoResolution">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentText">
    <string>1280_720</string>
   </property>
   <property name="currentIndex">
    <number>18</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>120_120   // 宽高比1:1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>160_160</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>270_270</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480_480</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>160_120   // 宽高比4:3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>240_180</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>280_210</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>320_240</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>400_300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480_360</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>640_480</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>960_720</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>160_90     // 宽高比16:9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256_144</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>320_180</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480_270</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>640_360</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>960_540</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280_720</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1920_1080</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>视频码率：</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>691</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="windowListBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
  </widget>
  <widget class="QListView" name="listView_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <zorder>listView_3</zorder>
  <zorder>listView_4</zorder>
  <zorder>listView_2</zorder>
  <zorder>listView</zorder>
  <zorder>addExcludedShareWindow</zorder>
  <zorder>videoFps</zorder>
  <zorder>pauseScreenCaptureBox</zorder>
  <zorder>selectScreenCaptureTarget</zorder>
  <zorder>removeAllExcludedShareWindow</zorder>
  <zorder>label_8</zorder>
  <zorder>right</zorder>
  <zorder>minVideoBitrate</zorder>
  <zorder>screenCaptureSourceType</zorder>
  <zorder>removeExcludedShareWindow</zorder>
  <zorder>label_10</zorder>
  <zorder>sdkVersionLabel_4</zorder>
  <zorder>enableHighLight</zorder>
  <zorder>label_11</zorder>
  <zorder>startScreenCaptureButton</zorder>
  <zorder>top</zorder>
  <zorder>enableCaptureMouse</zorder>
  <zorder>sdkVersionLabel_7</zorder>
  <zorder>screenCaptureSources</zorder>
  <zorder>sdkVersionLabel_3</zorder>
  <zorder>enableHighPerformance</zorder>
  <zorder>label_20</zorder>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>bottom</zorder>
  <zorder>left</zorder>
  <zorder>enableAdjustRes</zorder>
  <zorder>sdkVersionLabel_5</zorder>
  <zorder>videoBitrate</zorder>
  <zorder>label_21</zorder>
  <zorder>label_15</zorder>
  <zorder>resMode</zorder>
  <zorder>screenCaptureMixVolumeSlider</zorder>
  <zorder>label_16</zorder>
  <zorder>videoResolution</zorder>
  <zorder>label_9</zorder>
  <zorder>windowListBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
